src/types/transforms.js

```diff
@@ -251,7 +251,7 @@ export type TransformDefinitions = {
    *      ↓
    *      C
    */
-  'collapse-direction-recursion': {|
+  'collapse-direct-recursion': {|
```

Comment:
Small typo, but that isn't used anywhere. I only fixed it for consistency.

Reply:
Thanks for the fix. I wonder why we couldn't catch this with flow.

Reply:
I think that's because we never use this value, we use the `type`. It's here just to be nice.